adicionando combobox

DaviVeigaJardim opened the pull request.

DaviVeigaJardim committed:
adicionando combobox

DaviVeigaJardim committed:
retirando comando para inserir icone de play/pause

DaviVeigaJardim committed:
Criação dos botões reprodução aleatória e sequencial

Cria os botões reproduzir aleatoriamente e reproduzir sequencialmente

andre8359 commented on interfaceGráfica.py:
```diff
@@ -1,5 +1,6 @@
 from tkinter import ttk
+from tkinter import messagebox
 
 class Application:
```
nao misturar a nomeclatura, se for ter coisas em português colocar tudo em português

andre8359 commented on interfaceGráfica.py:
```diff
@@ -12,12 +13,12 @@ def __init__(self, master):
         
     def frames_da_tela(self):
```
quadros_da_tela

andre8359 commented on interfaceGráfica.py:
```diff
@@ -1,5 +1,6 @@
 from tkinter import *
 from tkinter import ttk
```
nunca colocar acentos ou espaços em nomes de arquivo

interfaceGrafica.py

DaviVeigaJardim committed:
Corrigindo nomes em ingles e adicionando função ao botão play

As funções com nomes em ingles foram retiradas e foi adicionado ao botão play a função que retorna uma mensagem dizendo "reproduzindo vídeo"

DaviVeigaJardim committed:
Erros corrigidos

DaviVeigaJardim committed:
Adicionando função ao botão de reprodução aleatória

andre8359 commented on interfaceGráfica.py:
```diff
@@ -1,40 +1,87 @@
-     self.frame_2 = Frame(self.root, bd=4, bg='#DDA0DD',
-                         highlightbackground='#FF69B4', highlightthickness=3)
-     self.frame_2.place(relx=0.02, rely=0.2, relwidth=0.96, relheight=0.76)
-    #Posições das cordenadas do frame de baixo, se o rely for igual, haverá sobreposição
```
self.quadro1
self.quadro2

andre8359 commented on Untitled-1.py:
```diff
@@ -0,0 +1,2 @@
+from pygame import *
+
```
por enquanto tu pode remover esse arquivo

andre8359 commented on interfaceGráfica.py:
```diff
@@ -1,40 +1,87 @@
+        self.combo_dias.bind("<<ComboboxSelected>>", self.selecionar_dia)
+
+        self.valores_mes = ['Janeiro', 'Fevereiro','março','abril','maio','junho','julho','agosto','setembro','outubro', 'novembro', 'dezembro']
+        self.combo_meses = ttk.Combobox(self.frame_1, values=self.valores_mes)
```
eu nao cheguei a olhar,  mas eu acho q essa função pega uma lista de str e não de int

DaviVeigaJardim committed:
Corrigindo nome para quadros e retirando acento do arquivo

DaviVeigaJardim committed:
tirando o segundo quadro e alterando cores e resolução
